### PR TITLE
[RF] New tutorial about bias and numeric issues in binned fits

### PR DESCRIPTION
**Pull request by guitargeek**
A new tutorial is added that explains how to deal with the bias and numerical instabilities in binned fits.

### CODE REVIEW

File: tutorials/roofit/rf614_bias_in_binned_fits.C [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+///  3. How to improve the numeric stability of fits with a greatly different
+///     number of events per bin, using a constant per-bin counterterm
+///
+/// \macro_code
+/// \macro_output
+///
+/// \date January 2023
+/// \author Jonas Rembser
+
+// Generate binned Asimuv dataset for a continuouts pdf.
```
Comment (lmoneta) [inline tutorials/roofit/rf614_bias_in_binned_fits.C:25]: Asimuv -> Asimov

Comment (lmoneta) [reply tutorials/roofit/rf614_bias_in_binned_fits.C:25]: Here also you are creating a data set with expected events. In generated binned the values for each bin will be generated according to a Poisson process. In principle you could add a gRandom->Poisson(nEvents * integ->getVal()) to have the correct result of `generatedBinned`

**Review by lmoneta — APPROVED** [review]
Thank you Jonas for the nice tutorial.
Maybe the tutorial could be split in two, one sowing the bias problem and the other showing the numerical stability.

File: tutorials/roofit/rf614_bias_in_binned_fits.C (hunk 1 above)
Comment (guitargeek) [reply tutorials/roofit/rf614_bias_in_binned_fits.C:25]: It is the intention that the dataset has the expected number of events. The `generateBinned()` function would do the same if you don't specify the `Extended()` command argument.

**Pull request merged**